### ISSUE TITLE
Delete mountpoint when tomb is closed

### ISSUE THREAD

**Issue by Bujiraso** [dyne/Tomb#149]
Currently the mountpoint of an open tomb is left behind upon closing the tomb. This could be considered a privacy leak, particularly when trying to hide a tomb amongst many files. It would be a good idea to delete this mountpoint directory after the tomb has closed.

**Comment by Bujiraso** [comment]
As of version 2.0.1, the intended functionality of deleting the auto-created mount-point is still not occurring for me. 
I notice this is for the 2.0 milestone, and @jaromil said that it was implemented over half a year ago.
Any thoughts?

**Comment by jaromil** [comment]
Thanks for watching this issue from close. I haven't yet run enough tests on it, but it looks like the new code that handles the deletion has introduced a problem while evolving to support the new /media/$USER mount default in some distributions. Here is the current code:

```
        # If we used a default mountpoint and is now empty, delete it
        [[ "$tombmount" -regex-match "[/run]?/media[/$_USER]?/$tombname.tomb" ]] && {
            rmdir $tombmount }
```

**Comment by jaromil** [comment]
Looks like this has broken since fix #180 as we forgot to keep a consistent naming scheme for mountpoints between the mount and umount (which hasn't removed the mandatory .tomb extension). This is the mount code right now:

```
    tombmount="$2"
    [[ "$tombmount" = "" ]] && {
        tombmount=/media/$TOMBFILE
        [[ -d /media ]] || { # no /media found, adopting /run/media/$USER (udisk2 compat)
            tombmount=/run/media/$_USER/$TOMBFILE
        }
        _message "Mountpoint not specified, using default: ::1 mount point::" $tombmount
    }
```

**Comment by jaromil** [comment]
Looking at the larger picture I believe all this is due because we haven't completed a recent refactoring: the `_plot()` function is now the central point for rendering the tomb path, filename and name, but is not being used everywhere (namely not in mount/umount operations).
